decidim-elections/app/views/decidim/elections/elections/show.html.erb

```diff
@@ -49,14 +49,42 @@ edit_link(
           <%= t(".vote") %>
         <% end %>
       <% else %>
-        <span class="title-action__action button small disabled"><%= t(".vote") %></span>
+        <span class="button disabled"><%= t(".vote") %></span>
       <% end %>
 
       <% if allowed_to? :preview, :election, election: election %>
-        <%= link_to new_election_vote_path(election.id), class: "title-action__action button small" do %>
+        <%= link_to new_election_vote_path(election.id), class: "button" do %>
           <%= t(".preview") %>
         <% end %>
       <% end %>
     </div>
   </div>
 </div>
+
+<!-- Election questions preview -->
+<div class="row section">
+  <div class="columns large-8">
+    <h2 class="section-heading"><%= t(".questions_preview.title") %></h2>
+    <p>
+      <%= t(".questions_preview.description") %>:
+    </p>
+    <ul class="accordion js-sortable mb-m evote__preview"
```

Comment:
Seems like the `.js-sortable` class is not used to style nor give some functionality, so I'd suggest removing it:

```suggestion
    <ul class="accordion mb-m evote__preview"
```